roosevelt.js

```diff
@@ -39,6 +39,7 @@ module.exports = function (params) {
   // expose initial vars
   app.set('express', express)
   app.set('params', params)
+  app.use('/scripts', express.static(params.appDir + '/node_modules/prismjs/'))
```

Comment:
This should be set in htmlValidator js in the same section where the page template is being pieced together.